includes/abstracts/abstract-batch.php

```diff
@@ -271,6 +278,19 @@ public function run( $current_step ) {
 		}
 
 		$progress = ( 0 === (int) $total_steps ) ? 100 : round( ( $this->current_step / $total_steps ) * 100 );
+
+		// If there are errors, return success false.
+		// Next step would be to pass the errors down and display them with JS.
```

Comment:
I can't tell if this comment is a leftover `// todo` type of comment or if it is intentional. Is it helpful here?